source/manual/email-notifications-how-they-work.html.md

```diff
@@ -13,6 +13,20 @@ The purpose of the email notifications system is to inform users when content th
 
 **Update September 2019:** the email system also now supports highly customised subscriptions for users completing the [Brexit Checker][brexit-checker], which operates in isolation from the rest of GOV.UK, and specifies its own notifications. More information can be found in [the ADR][message-adr] for these changes.
 
+## Types of email
+
+The email notification system generates 3 different types of email. These are transactional, immediate and digest.
+
+**Transactional** email allow an individual user to work with the email notification system. Examples of these are forgotten password links, subscription confirmations, authentication links. These are sent as the result of a user interaction with the email system
+
+**Immediate** emails are generated shortly the email system receives a content change, or a message, that corresponds to a users immediate subscription. It's worth noting that "immediate" is a somewhat misleading name as these emails can sometimes take a long time (hours) to be sent when the system is busy.
+
+**Digest** emails are generated daily and weekly for subscribers who have opted to have their subscriptions sent to them in batches. These collate all of a subscribers subscriptions into a single email including all the relevant content changes and messages that occurred over the preceding day or week.
+
+### Courtesy copies
```

Comment:
This isn't really a distinct type of email, but would fit quite well in the next-but-one section if we renamed "Email system apps" -> "System components" e.g.

```
- Google Groups
  - Provides a way to see [courtesy copies][google-group] of emails sent in production.
  - Provides a way to test emails in [Integration and Staging][link-to-doc]
```

Reply:
Sure, that's a good idea - I like that is concise. I think it might actually fit better in useful resources, which avoids the rename complication.